src/services/SnapshotService.ts

```diff
@@ -97,4 +98,11 @@ export class SnapshotService {
     }
     return { snapshotStatus, snapshotSpace }
   }
+
+  static async getAddressesVotes(addresses: string[], first?: number, skip?: number) {
+    if (isNumber(first) && isNumber(skip)) {
```

Comment:
I think you also have native options like Number.isInteger

Reply:
Yes but the compiler was less happy with it